tests/baselines/reference/pushTypeGetTypeOfAlias.errors.txt

```diff
@@ -0,0 +1,26 @@
+error TS5055: Cannot write file 'tests/cases/compiler/bar.js' because it would overwrite input file.
+  Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
+error TS5055: Cannot write file 'tests/cases/compiler/foo.js' because it would overwrite input file.
+  Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
+tests/cases/compiler/bar.js(2,2): error TS2303: Circular definition of import alias 'blah'.
+tests/cases/compiler/bar.js(2,25): error TS2339: Property 'unknown' does not exist on type '{ (): void; blah: any; }'.
+tests/cases/compiler/foo.js(2,5): error TS2339: Property 'nonexistentProperty' does not exist on type '{ (): void; blah: any; }'.
+
+
+!!! error TS5055: Cannot write file 'tests/cases/compiler/bar.js' because it would overwrite input file.
+!!! error TS5055:   Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
+!!! error TS5055: Cannot write file 'tests/cases/compiler/foo.js' because it would overwrite input file.
+!!! error TS5055:   Adding a tsconfig.json file will help organize projects that contain both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig.
+==== tests/cases/compiler/foo.js (1 errors) ====
+    var bar = require("./bar");
+    bar.nonexistentProperty;
+        ~~~~~~~~~~~~~~~~~~~
+!!! error TS2339: Property 'nonexistentProperty' does not exist on type '{ (): void; blah: any; }'.
+==== tests/cases/compiler/bar.js (2 errors) ====
+    module.exports = function () {};
+     exports.blah = exports.unknown;
+     ~~~~~~~~~~~~
+!!! error TS2303: Circular definition of import alias 'blah'.

```

Comment:
This circular definition error is misleading, but I think that's a widespread problem with any use of our current push/popTypeResolution infrastructure. We should go solve it as some point, though.